content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -209,7 +209,8 @@ described [above](#deployment-and-scaling-guarantees).
 `Parallel` pod management tells the StatefulSet controller to launch or
 terminate all Pods in parallel, and to not wait for Pods to become Running
 and Ready or completely terminated prior to launching or terminating another
-Pod.
+Pod. This option only affects the behavior for scaling operations, updates are not
```

Comment:
Comma splice.

Either "This option only affects the behavior for scaling operations; updates are not affected."
or "This option only affects the behavior for scaling operations. Updates are not affected."

Reply:
I went with the latter